src/main/java/spark/http/matching/MatcherFilter.java

```diff
@@ -149,7 +149,8 @@ public void doFilter(ServletRequest servletRequest,
         }
 
         if (body.notSet() && !externalContainer) {
-            LOG.info("The requested route [" + uri + "] has not been mapped in Spark");
+            LOG.info("The requested route [{}] has not been mapped in Spark for {}: [{}]",
+                uri, ACCEPT_TYPE_REQUEST_MIME_HEADER, acceptType);
```

Comment:
It might be clearer to only print the bit about the accept header if one was actually provided.